Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Core\ArrayTrait;
+use Google\Cloud\Core\ClientTrait;
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient as InstanceAdminClient;
+use Google\Cloud\Bigtable\Connection\Grpc;
+use Google\Cloud\Bigtable\Connection\LongRunningConnection;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+
+/**
+ * Cloud Bigtable is a highly scalable, transactional, managed, NewSQL
+ * database service. Find more information at
+ * [Cloud Bigtable docs](https://cloud.google.com/bigtable/).
+ *
+ * In production environments, it is highly recommended that you make use of the
+ * Protobuf PHP extension for improved performance. Protobuf can be installed
+ * via [PECL](https://pecl.php.net).
+ *
+ * ```
+ * $ pecl install protobuf
+ * ```
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Bigtable\BigtableClient;
+ *
+ * $bigtable = new BigtableClient();
+ * ```
+ *
+ * @method resumeOperation() {
+ *     Resume a Long Running Operation
+ *
+ *     Example:
+ *     ```
+ *     $operation = $bigtable->resumeOperation($operationName);
+ *     ```
+ *
+ *     @param string $operationName The Long Running Operation name.
+ *     @param array $info [optional] The operation data.
+ *     @return LongRunningOperation
+ * }
+ */
+class BigtableClient
+{
+    use ArrayTrait;
+    use ClientTrait;
+    use LROTrait;
+
+    const FULL_CONTROL_SCOPE = 'https://www.googleapis.com/auth/bigtable.data';
+    const ADMIN_SCOPE = 'https://www.googleapis.com/auth/bigtable.admin';
+
+    /**
+     * @var Connection\ConnectionInterface
+     */
+    protected $connection;
+
+    /**
+     * Create a Bigtable client. Please note that this client requires
+     * [the gRPC extension](https://cloud.google.com/php/grpc).
+     *
+     * @param array $config [optional] {
+     *     Configuration Options.
+     *
+     *     @type string $projectId The project ID from the Google Developer's
+     *           Console.
+     *     @type array $authCacheOptions Cache configuration options.
```

Comment:
Is this needed after we remove $authCache?
In fact, why do we remove this while we're still needing $authCacheOptions and $credentialsFetcher?
@jdpedrie  Could you confirm too?

Reply:
`$authCache`, `$authCacheOptions`, and `$credentialsFetcher` all still apply and should not be removed.